### PR TITLE
🤖 Simplify `hello-world` stub

### PR DESCRIPTION
**Pull request by ErikSchierboom**
Simplify the `hello-world` exercise's stub. The goal of this is to make things easier for students to get started and to introduce as little syntax as possible.

See https://github.com/exercism/v3-launch/issues/46

### CODE REVIEW

**Review by mpizenberg — COMMENTED** [review]
That's cleaner indeed! and the "Goodby, Mars!" makes it clear what they have to change. We probably don't need the type annotation though since they are not mandatory and introduced in the second concept exercise. What made you feel we should add the annotation?

**Comment by ErikSchierboom** [comment]
@mpizenberg Oh no real reason, I just used the example file as the basis, and that apparently includes the annotation. We should definitely lose the annotation if it is not needed. I'll update!

File: exercises/practice/hello-world/src/HelloWorld.elm [hunk 1]
```diff
@@ -1,22 +1,6 @@
...
-
-   The module name is expected by the test program and must match the name of this
-   file. It has to stay just the way it is.
--}
-
-
 module HelloWorld exposing (helloWorld)
 
 
+helloWorld : String
```
Comment (ErikSchierboom) [inline exercises/practice/hello-world/src/HelloWorld.elm:4]:
```suggestion
```

**Review by mpizenberg — APPROVED** [review]
Awesome, thanks @ErikSchierboom !

**Pull request merged**